prow/plugins.yaml

```diff
@@ -354,3 +354,7 @@ external_plugins:
   - name: needs-rebase
     events:
       - pull_request
+  kubernetes/kubernetes:
+  - name: needs-rebase
+    events:
+      - pull_request
```

Comment:
Why don't external plugins declare their own events?

Reply:
We could add that, but instead of blindly dispatching hooks to external plugins as we do now, we would have to create a registration phase that determines what events to send to an external plugin and periodically re-register in case the event requirements for a plugin changed.
Alternatively, we could allow external plugins to be defined once in the config and then just use the plugin name to enable the plugin on various orgs and repos without repeating the event requirements or endpoint.
This outside the scope of this PR though.

Reply:
Yeah, or rearrange like:

    external_plugins:
      needs-rebase:
        events:
        - pull_request
        repos:
        - kubernetes/kubernetes
        - kubernetes/kops
       ....

Doesn't need to be in this PR.

Reply:
Yeah, I also prefer to rearrange the config for the external plugins. IIRC the argument against it was that it is different from the normal plugin config.